src/assessment.handlebars

```diff
@@ -1041,8 +1043,9 @@
   <div class="slide main-menu" id="main-menu">
     <div class="flow">
       <h1 tabindex="-1">ESA Assessment</h1>
-      <p>If you can&#39;t work because of illness or disability, you could be eligible for Employment and Support Allowance &#40;<abbr title="Employment And Support Allowance">ESA</abbr>&#41;.</p>
+      <p>If you can&#39;t work because of illness or disability, you could be eligible for Employment and Support Allowance &#40;<abbr title="Employment And Support Allowance">ESA</abbr>&#41;.</p> 
       <p>We understand that the benefits system can be confusing, so this site is here to help you prepare for the assessment.</p>
+      <p>In some areas Universal Credit is replacing new claims for ESA, for more information about universal credit visit https://www.gov.uk/universal-credit and https://ucpostcode.entitledto.co.uk/ucdate to find out if you need to claim UC or ESA.  In both cases the questions and scoring on this site will apply</p>
```

Comment:
There might be an extra space before the final sentence and it is missing a fullstop at the end.

Reply:
Should the urls be wrapped in <a>?